Samples/Samples/ViewModel/LauncherViewModel.cs

```diff
@@ -13,17 +12,33 @@ public class LauncherViewModel : BaseViewModel
 
         public ICommand CanLaunchCommand { get; }
 
+        public ICommand LaunchMailCommand { get; }
+
+        public ICommand LaunchBrowserCommand { get; }
+
         public LauncherViewModel()
         {
             LaunchCommand = new Command(OnLaunch);
+            LaunchMailCommand = new Command(OnLaunchMail);
+            LaunchBrowserCommand = new Command(OnLaunchBrowser);
             CanLaunchCommand = new Command(CanLaunch);
         }
 
+        async void OnLaunchBrowser()
+        {
+            await Launcher.OpenAsync("https://github.com/xamarin/Essentials");
+        }
+
         async void OnLaunch()
         {
             await Launcher.OpenAsync(LaunchUri);
         }
 
+        async void OnLaunchMail()
+        {
+            await Launcher.OpenAsync("mailto:");
```

Comment:
Is it possible that these throw an exception at all? Perhaps we should create a generic method that tries to launch it and then handle the exception here.

Reply:
It is a valid uri, so there will be no exception. iOS definitely has a mail app, uwp does too. Android has some mail app 99% of the time, you could probably have no mail app instaled if you use a gApps free os. Either way, no exception would be thrown and the intent would just be unhandled.